sdk/nodejs/appservice/zMixins_http.ts

```diff
@@ -32,6 +32,34 @@ export type HttpRequest = azurefunctions.HttpRequest;
  */
 export type HttpResponse = azureessentials.HttpResponse;
 
+/**
+ * Host settings specific to the HTTP plugin.
+ *
+ * For more details see https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json#http
+ */
+export interface HttpHostSettings extends mod.HostSettings {
+    extensions?: {
+        http: {

```

Comment:
would it make sense to have this be optional too?  or are we really saying "if you provide 'extensions' is *must* have an 'http' property in it"?

Reply:
For now, it's going to be the only property, so makes sense to require it IMO. Making it optional later shouldn't break anything, right?

Reply:
it technically can.  but it's highly unlikely.  so i'm ok with it being required.